libs/article/api/src/lib/article.service.ts

```diff
@@ -194,6 +194,14 @@ export class ArticleService {
         hidden,
         disableComments,
         revisions: {
+          updateMany: {
```

Comment:
When creating a new revision, update the old ones to be archived.

<img width="939" alt="image" src="https://github.com/user-attachments/assets/520fdd44-11db-4f6b-afc3-e067593ab0a1" />

Made sure this works in the correct order.